docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -1431,8 +1462,10 @@ spring:
           resource-group: ${SERVICEBUS_RESOURCE_GROUP}
 ----
 
-To use the security principal for authentication directly, taking service principals as example, users could use below configurations, where the principal should
-be granted with necessary `Data` related roles for messaging operations.
+For authentication based on Azure Active Directory, security principals of the service principal and the manged identity are both supported, where the principal should
+be granted with necessary `Data` related roles for messaging operations. The configuration examples of the service principal and the manged identity are listed as below:
```

Comment:
You can also migrate to authenticate and authorize with Azure AD directly without making a detour to ARM. 
- With a service principal
- With a managed identity